app/menu.js

```diff
@@ -132,6 +132,13 @@ module.exports = class MenuBuilder {
           click: () => {
             this.mainWindow.setFullScreen(!this.mainWindow.isFullScreen());
           }
+        },
+        {
+          label: 'Toggle Developer Tools',
```

Comment:
devtools is useful for grpc-web to inspect network tab, but shortcut should use system